egs/ljspeech/TTS/prepare.sh

```diff
@@ -80,6 +80,10 @@ fi
 
 if [ $stage -le 3 ] && [ $stop_stage -ge 3 ]; then
   log "Stage 3: Prepare phoneme tokens for LJSpeech"
+  # We assume you have installed piper_phonemize and espnet_tts_frontend.
+  # If not, please install them with:
+  #   - piper_phonemize: refer to https://github.com/rhasspy/piper-phonemize
```

Comment:
Please also add the link to pre-built wheels
https://github.com/csukuangfj/piper-phonemize/releases/tag/2023.12.5